operations/update.go

```diff
@@ -53,12 +53,22 @@ func Update() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
-			return DoUpdate(conf, ctx, doInstall, forceUpdate)
+			if conf.AutoUpgradeCLI {
```

Comment:
I don't think this block should be necessary since `CheckAndUpdateVersion` already checks if the command is really up-to-date or not. There would be some benefit in being told that they have the most up-to-date version even if auto-upgrading is enabled. In this case, I would let `evergreen get-update` say the "Binary is already up to date - not updating." message to let them explicitly know they're on the newest version.